src/components/Table/CardTable/index.vue

```diff
@@ -248,6 +248,10 @@ export default {
     display: flex;
     justify-content: left;
     flex-wrap: wrap;
+
+    .el-card .el-card__body div {
+      height: inherit;
+    }
   }
 
   .el-col, div {
```

Comment:
I'm sorry for any misunderstanding but as I am written in July 2025, my knowledge is solely at that time point. If you have specific codes for comparison, please provide them and I would be glad to help with any differences or suggestions for improvement!